scripts/wail.coffee

```diff
@@ -14,14 +14,22 @@
 #   csoni111
 
 module.exports = (robot) ->
-  robot.respond /who.*lab/i, (msg) ->
-    waliUrl = process.env.WAIL_PIC_URL + '?t=' + new Date().getTime()
-    msg.send(
-      "attachments": [
+    robot.respond /who.*lab/i, (msg) ->
+        waliUrl = process.env.WAIL_PIC_URL + '?t=' + new Date().getTime()
+        wailTextUrl = process.env.WAIL_NAME_URL + '?t=' + new Date().getTime()
+        
+        request = require 'request'
+
+        resp = ""
+        request.get {uri:'#{wailTextUrl}'}, (err, r, body) ->
+            resp = body
+
+        msg.send(
+            "attachments": [
             "fallback": "Here's a pic: #{waliUrl}"
             "color": "#36a64f"
-            "pretext": "Here's a pic:"
+            "pretext": "#{resp} \n Here's a pic:"
```

Comment:
Add the complete message to `fallback` too. In case the attachment message fails, slack displays the fallback text.

Reply:
when it fails, will the response be null?